plot/plot.grammar

```diff
@@ -18,6 +18,7 @@
 # generate candidates from the paths, should be floating
 (rule $PathPattern (anypath) (ConstantFn *) (floating 1))
 (rule $Value (anyvalue) (ConstantFn *) (floating 1))
+(rule $Value (anyvalue) (interactive.JsonFn ConstantValue) (floating 1))
```

Comment:
doing this at the grammar level is better and cleaner, but the previous constant values still seem to be at VegaResources.java L176

Reply:
Ok, I'll remove them from VegaResources.  btw, any reason 100 is a special number?

Reply:
No. It used to be 42. Originally thought that the number 100 can trigger some interesting properties (as in 100% of something), but in reality it didn't.

Reply:
no, just needed to pull some number out. no reason to prefer one over another, except that 0 might be a bit different, and almost not a number.
an issue with removing is that VegaRandomizer might not take this path (unverified)

Reply:
we could return some special value, and let the client set via an input/slider in the future

Reply:
hmm yeah I see the issue with VegaRandomizer...is it that bad if we just leave VegaResources.java as is?